src/cdk-experimental/scrolling/auto-size-virtual-scroll.ts

```diff
@@ -85,6 +85,9 @@ export class AutoSizeVirtualScrollStrategy implements VirtualScrollStrategy {
   /** The last measured size of the rendered content in the viewport. */
   private _lastRenderedContentOffset: number;
 
+  /** The number of consecutive cycles where removing extra items has failed. */
```

Comment:
Would it make sense to explain what a render failure means? E.g. is it an exception being thrown, not being able to measure how high the element is etc.